tests/integration/test_requirements_dev_version_constraints.py

```
@@ -0,0 +1,280 @@
+    
+    def test_version_constraints_parseable(self, req_lines: List[str]):
+        """
+        Validate that every requirement line contains a parseable version specifier.
+        
+        Each line is split into a package name and a version specification and the version specification is validated using packaging.specifiers.SpecifierSet; the test fails if any specifier is invalid.
+        """
+        for line in req_lines:
+            match = re.match(r'^([a-zA-Z0-9_\-\[\]]+)(.*)$', line)
+            assert match, f"Cannot parse requirement line: {line}"
+            
+            pkg_name, version_spec = match.groups()
```

Ruff (0.14.6)

Unpacked variable `pkg_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)